src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -449,6 +449,20 @@ void assertContainsChildOfType(Collection<Class<? extends Element>> elementTypes
     }
   }
 
+  /**
+   * Assert that children with any of the informed types do not exceed the maximum count allowed.
+   *
+   * @param elementTypes The element types that will be verified in order to ensure that the maximum count was not exceed.
+   * @throws InvalidInputException
+   */
+  void assertChildrenNotExceedingMaxCount(Collection<Class<? extends Element>> elementTypes, int maxCountPerElementType) throws InvalidInputException {
+    boolean hasExceeded = elementTypes.stream().anyMatch(type -> findElements(type).size() > maxCountPerElementType);
+    if (hasExceeded) {
+      throw new InvalidInputException(String.format("Element \"form\" cannot have more than %s children of the following elements: [%s].",
```

Comment:
I just realized now, but you say "Element "form" cannot have" when the element is the current element, which is not necessarily the form.